pkg/ccl/backupccl/targets.go

```diff
@@ -209,6 +209,56 @@ func getAllDescChanges(
 	return res, nil
 }
 
+// blockMultiRegionTableBackup validates that for all tables included in the
+// backup, their parent database is also being backed up.
+func blockMultiRegionTableBackup(
+	backupStmt *annotatedBackupStatement,
+	descs []catalog.Descriptor,
+	tables []catalog.TableDescriptor,
+) error {
+	// We only need to block in the table backup case, so there's nothing to do
+	// if we're running a cluster backup.
+	if backupStmt.Coverage() == tree.AllDescriptors {
+		return nil
+	}
+	// We build a map of the target databases here because the supplied list of
+	// descriptors contains ALL database descriptors for the corresponding
+	// tables (regardless of whether or not the databases are included in the
+	// backup targets list). The map helps below so that we're not looping over
+	// the descriptors slice for every table.
+	databaseTargets := map[descpb.ID]struct{}{}
+	for _, desc := range descs {
+		switch desc.(type) {
+		case catalog.DatabaseDescriptor:
+			// If the database descriptor found is included in the targets list, add
+			// it to the map.
+			for _, name := range backupStmt.Targets.Databases {
+				if desc.GetName() == name.String() {
```

Comment:
i think `name.String()` will escape the database name. should this be `tree.Name(desc.GetName()) == name.String()`? can you double check with a test? i may or may not be taking crazy pills